utils/src/server/rpc_module.rs

```diff
@@ -27,7 +27,7 @@ pub type SubscriptionId = u64;
 /// Sink that is used to send back the result to the server for a specific method.
 pub type MethodSink = mpsc::UnboundedSender<String>;
 
-type Subscribers = Arc<Mutex<FxHashMap<(ConnectionId, SubscriptionId), MethodSink>>>;
+type Subscribers<P> = Arc<Mutex<FxHashMap<(ConnectionId, SubscriptionId), InnerSink<P>>>>;
```

Comment:
Should this be `pub`? I can't think of a good reason right now, but it feels like maybe it should?

Reply:
No, it's just an internal type so no need to make it `pub` yet.